src/datasource.js

```diff
@@ -77,9 +78,12 @@ export class BitmovinAnalyticsDatasource {
       }
       data['groupBy'] = target.groupBy;
       data['limit'] = Number(target.limit) || undefined;
-
+      var apiRequestUrl = this.url + '/analytics/queries';
+      if (this.isAdAnalytics === true) {
```

Comment:
move this to a method. doing two `var apiRequestUrl` is bad form

Reply:
Moved to method